### DISCUSSION TITLE
Reading Victor B sides

### DISCUSSION THREAD

**Discussion by hanshuebner** [davidgiven/fluxengine#793] [Q&A] [answered]
Hi,

I'm working on archiving a box of Victor floppies.  Initially, I tried reading with an NEC FD1155C with little success - The inner tracks consistently read with errors.  I then tried a Hitachi HFD516CI2 from an IBM 3174 terminal concentrator, and amazingly, that drive works much better and reads many of the floppies with no errors at all.

Unfortunately, the floppies are written single sided, but many of them on both sides.  Fluxengine refuses to read the B sides because it does not get index pulses.  I could certainly try to cut a hole into the sleeve, but I'd really like to avoid that.  Is there a way to define a format for the B side which would basically have to revert the flux data before decoding it?  That way, I could read the B sides using the second head, with the floppy inserted as normal.

Any other ideas how to get to the B side data would be greatly appreciated.

Thanks!
Hans

**Comment by hanshuebner** [comment, marked as answer]
I learned that reading the B side won't work because the tracks are not arranged symetrically on the front and back side.  As I only have a couple of flippy floppies to read, I'll probably make extra holes or flip the medium in the sleeve to read them.

Thanks for reading!
Hans

**Comment by davidgiven** [comment]
Yup, that's correct.

The biggest problem is less that FluxEngine needs the index pulses and more that the drive itself won't enables output data until the index pulses tell it the disk is up to speed. One workaround is simply to spoof the pulses directly to the drive. The FluxEngine hardware provides some useful outputs for this. See the section on flippy disks here: http://cowlark.com/fluxengine/doc/faq.html
